pytest_freezer.py

```diff
@@ -1,9 +1,6 @@
-"""Pytest plugin providing a fixture interface for spulec/freezegun"""
 import pytest
 import freezegun
 
-__version__ = "0.4.2"
```

Comment:
You can still do that https://hatch.pypa.io/latest/version/

Reply:
Sure, but with `importlib.metadata` no longer provisional it seems redundant to have the version string in both the metadata and the module namespace. So I just keep it once in the `pyproject.toml` now.